GEOSldas_App/ldas_setup

```diff
@@ -504,21 +579,28 @@ class LDASsetup:
 
         inpDictFromFile = self._parseInputFile(cmdLineArgs['batinpfile'])
 
-        # REQUIRED inputs
-        for key in rqdRmInpKeys:
-            self.rqdRmInp[key] = inpDictFromFile.pop(key)
-
-        # checks on rqd rm inputs
-        ## account and walltime should exist
-        assert self.rqdRmInp['account']
-        if cmdLineArgs['account'] != 'None':
-           self.rqdRmInp['account'] = cmdLineArgs['account']
-        assert self.rqdRmInp['walltime']
-        ## ntasks_model is a +ve integer
-        _ntasks = int(self.rqdRmInp['ntasks_model'])
-        assert _ntasks>0
-        self.rqdRmInp['ntasks_model'] = _ntasks
-        _ntasks = None
+        if self.ladas_cpl > 0 : 
+           if cmdLineArgs['account'] != 'None':
+                self.rqdRmInp['account'] = cmdLineArgs['account']
+           self.rqdRmInp['walltime']        = "00:30:00" 
```

Comment:
@saraqzhang : Are we sure that 30 min is sufficient for c360 and, eventually, c720 or higher?  Maybe increase to 1 hour?